ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -50,14 +50,14 @@ Item {
 
         readonly property real scrollY: chatLogView.visibleArea.yPosition * chatLogView.contentHeight
         readonly property bool isMostRecentMessageInViewport: chatLogView.visibleArea.yPosition >= 0.999 - chatLogView.visibleArea.heightRatio
-        readonly property var chatDetails: chatContentModule.chatDetails
+        readonly property var chatDetails: chatContentModule.chatDetails || null
```

Comment:
Why do we need `|| null` here ? :thinking:

Reply:
fixed some warning